public_html/css/sass/_chat.scss

```diff
@@ -0,0 +1,53 @@
+.chat-view {
+  bottom: 0;
+  font-size: 12px;
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.chat {
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.